data/blog/electron-6-0.md

```diff
@@ -0,0 +1,122 @@
+
+These functions now return Promises and still support older callback-based invocation:
+ * contentTracing.getCategories() [#16583](https://github.com/electron/electron/pull/16583)
+ * contentTracing.getCategories() [#16583](https://github.com/electron/electron/pull/16583)
+ * contentTracing.getTraceBufferUsage() [#16600](https://github.com/electron/electron/pull/16600)
+ * contents.executeJavaScript() [#17312](https://github.com/electron/electron/pull/17312)
+ * cookies.flushStore() [#16464](https://github.com/electron/electron/pull/16464)
+ * cookies.get() [#16464](https://github.com/electron/electron/pull/16464)
+ * cookies.remove() [#16464](https://github.com/electron/electron/pull/16464)
+ * cookies.set() [#16464](https://github.com/electron/electron/pull/16464)
+ * dialog.showCertificateTrustDialog() [#17181](https://github.com/electron/electron/pull/17181)
+ * inAppPurchase.getProducts() [#17355](https://github.com/electron/electron/pull/17355)
+ * inAppPurchase.purchaseProduct()[#17355](https://github.com/electron/electron/pull/17355)
+ * netLog.stopLogging() [#16862](https://github.com/electron/electron/pull/16862)
+ * session.clearAuthCache() [#17259](https://github.com/electron/electron/pull/17259)
+ * session.clearCache()  [#17185](https://github.com/electron/electron/pull/17185)
+ * session.clearHostResolverCache() [#17229](https://github.com/electron/electron/pull/17229)
+ * session.clearStorageData() [#17249](https://github.com/electron/electron/pull/17249)
+ * session.getBlobData() [#17303](https://github.com/electron/electron/pull/17303)
+ * session.getCacheSize()  [#17185](https://github.com/electron/electron/pull/17185)
+ * session.resolveProxy() [#17222](https://github.com/electron/electron/pull/17222)
+ * session.setProxy()  [#17222](https://github.com/electron/electron/pull/17222)
+ * webContents.hasServiceWorker() [#16535](https://github.com/electron/electron/pull/16535)
+ * webContents.printToPDF() [#16795](https://github.com/electron/electron/pull/16795)
+ * webContents.savePage() [#16742](https://github.com/electron/electron/pull/16742)
+ * webFrame.executeJavaScript() [#17312](https://github.com/electron/electron/pull/17312)
+ * webFrame.executeJavaScriptInIsolatedWorld() [#17312](https://github.com/electron/electron/pull/17312)
+ * webviewTag.executeJavaScript() [#17312](https://github.com/electron/electron/pull/17312)
+
+These functions now have two forms, synchronous and Promise-based asynchronous:
+ * `dialog.showMessageBox()` [#17298](https://github.com/electron/electron/pull/17298)
+ * `dialog.showOpenDialog()` [#16973](https://github.com/electron/electron/pull/16973)
+ * `dialog.showSaveDialog()` [#17054](https://github.com/electron/electron/pull/17054)
+
+These functions now return Promises:
+ * `app.dock.show()` [#16904](https://github.com/electron/electron/pull/16904)
+
+### `Electron Helper (Render).app` and `Electron Helper (Plugin).app`
```

Comment:
We should add a new block here indicating how folks should support these helpers when packaging .  Something like

```
In order to package your application correctly with these two new helpers you need to be using `electron-packager@14.0.3` or higher.  If you are using `electron-builder` you should follow [this issue](https://github.com/electron-userland/electron-builder/issues/4104) to track support for these new helpers.
```